server/lib/tuist_web/components/layouts/ops.html.heex

```diff
@@ -39,6 +39,13 @@
               navigate={~p"/ops/sent_emails"}
             />
           <% end %>
+          <%= if not Tuist.Environment.dev_use_remote_storage?() do %>
+            <.tab_menu_horizontal_item
+              label={gettext("Storage")}
+              target="_blank"
+              navigate={"http://localhost:#{Tuist.Environment.minio_console_port()}"}
```

Comment:
@cschmatzler I couldn't get [`href`](https://github.com/tuist/Noora/blob/153ca853d30889943dfdd3aa53648779d691c152/web/lib/noora/tab_menu.ex#L105) working. Shouldn't we handle that case in the control flow too?